Dockerfile

```diff
@@ -41,6 +41,11 @@ RUN apt-get update && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
+# Make link to libnvidia-ml.so (NVML) library
+# so that we could get GPU stats.
+RUN ln -s /usr/lib/x86_64-linux-gnu/libnvidia-ml.so.1 \
```

Comment:
What error it will generate if not adding this line?

Reply:
`libnvidia-ml.so: cannot open shared object file: No such file or directory`